gcc/config.gcc

```diff
@@ -1264,24 +1264,40 @@ aarch64*-*-linux*)
 	TM_MULTILIB_CONFIG=`echo $TM_MULTILIB_CONFIG | sed 's/^,//'`
 	;;
 aarch64*-*-mingw*)
-# 	tm_file="${tm_file} dbxelf.h elfos.h gnu-user.h linux.h glibc-stdint.h"
-# 	tm_file="${tm_file} aarch64/aarch64-elf.h aarch64/aarch64-errata.h aarch64/aarch64-linux.h"
-# 	tm_file="${tm_file} i386/unix.h i386/bsd.h i386/gas.h dbxcoff.h i386/cygming.h"	
-	tm_file="${tm_file} aarch64/aarch64-coff.h"
-	tm_file="${tm_file} aarch64/cygming.h"
+	tm_file="${tm_file} aarch64/aarch64-coff.h aarch64/cygming.h" # i386/unix.h i386/bsd.h i386/gas.h
+	xm_file=i386/xm-mingw32.h
+	c_target_objs="${c_target_objs} winnt-c.o"
+	cxx_target_objs="${cxx_target_objs} winnt-c.o"
+	d_target_objs="${d_target_objs} winnt-d.o"
+	target_has_targetcm="yes"
+	target_has_targetdm="yes"
+	#rust_target_objs="${rust_target_objs} winnt-rust.o"
```

Comment:
not sure we need rust support

Reply:
Not yet, it's commented out anyway. I left it here for easier comparison with `x86_64-w64-mingw32` sections in case it will be changing upstream before the corresponding changes in the `woarm64` will be merged upsteam.